src/main/java/org/apache/xmlbeans/impl/tool/MavenPlugin.java

```diff
@@ -15,6 +15,13 @@
 
 package org.apache.xmlbeans.impl.tool;
 
+import org.apache.commons.io.FileUtils;
```

Comment:
we don't yet have a compile/runtime dependency on the commons-io jar

commons-io is only currently used in some tests

The POM and build files will need to be changed

It would probably also require us to rename the next release as 5.3.0 because the extra dependenecy makes this a less trivial upgrade.